scapy/contrib/bgp.py

```diff
@@ -2185,8 +2213,10 @@ class BGPUpdate(BGP):
     ]
 
     def post_build(self, p, pay):
+        # type: (bytes, bytes) -> bytes
         subpacklen = lambda p: len(p)
-        packet = ""
```

Comment:
good catch. Could you add a unit test?
https://github.com/secdev/scapy/tree/master/test/contrib/bgp.uts